txscript/standard_test.go

```diff
@@ -568,26 +573,48 @@ func TestPayToAddrScript(t *testing.T) {
 	}
 
 	//  mainnet p2pk 13CG6SJ3yHUXo4Cr2RY4THLLJrNFuG3gUg
-	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKey(decodeHex("02192d74"+
-		"d0cb94344c9569c2e77901573d8d7903c3ebec3a957724895dca52c6b4"),
+	pubkey1, err := chainec.Secp256k1.ParsePubKey(decodeHex("02192d74" +
+		"d0cb94344c9569c2e77901573d8d7903c3ebec3a957724895dca52c6b4"))
+
+	if err != nil {
+		t.Errorf("Unable to parse pubkey (compressed): %v",
+			err)
+		return
+	}
+	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKeyCompressed(pubkey1,
 		&chaincfg.MainNetParams)
 	if err != nil {
 		t.Errorf("Unable to create pubkey address (compressed): %v",
 			err)
 		return
 	}
-	p2pkCompressed2Main, err := dcrutil.NewAddressSecpPubKey(decodeHex("03b0bd"+
-		"634234abbb1ba1e986e884185c61cf43e001f9137f23c2c409273eb16e65"),
+
+	pubkey2, err := chainec.Secp256k1.ParsePubKey(decodeHex("03b0bd" +
+		"634234abbb1ba1e986e884185c61cf43e001f9137f23c2c409273eb16e65"))
+
+	if err != nil {
+		t.Errorf("Unable to parse pubkey (compressed 2): %v",
+			err)
+		return
+	}
+	p2pkCompressed2Main, err := dcrutil.NewAddressSecpPubKeyCompressed(pubkey2,
 		&chaincfg.MainNetParams)
 	if err != nil {
 		t.Errorf("Unable to create pubkey address (compressed 2): %v",
 			err)
 		return
 	}
 
-	p2pkUncompressedMain, err := dcrutil.NewAddressSecpPubKey(decodeHex("0411db"+
```

Comment:
This could just use the `newAddressPubKey` function.


```Go
p2pkUncompressedMain := newAddressPubKey(decodeHex(....),
	&chaincfg.MainNetParams
```